pkgs/by-name/ne/nerdfetch/package.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitea
+}:
+stdenv.mkDerivation rec {
+  pname = "nerdfetch";
+  version = "8.1.0";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "thatonecalculator";
+    repo = "NerdFetch";
+    rev = "v${version}";
+    hash = "sha256-yV1IYUixBkiFEB2NfKdvK/8XT0BrQ5XsBME7XWNDXZU=";
+  };
+
+  dontUnpack = true;
+  dontConfigure = true;
+  dontBuild = true;
+  installPhase = ''
+    mkdir -p $out/bin
+      cp $src/nerdfetch $out/bin
+  '';
+
+  meta = with lib;{
+    description = "A POSIX *nix (Linux, macOS, Android, *BSD, etc) fetch script using Nerdfonts";
+    homepage = "https://codeberg.org/thatonecalculator/nerdfetch";
+    maintainers = with maintainers; [ ByteSudoer ];
+    license = licenses.mit;
+    mainProgram = "nerdfetch";
+  };
+}
```

See above regarding switch to finalAttrs
```suggestion
})
```